lib/symbol_to_status_code.rb

```diff
@@ -0,0 +1,62 @@
+# Inspired from
+# http://www.rubydoc.info/github/rack/rack/master/Rack/Utils#HTTP_STATUS_CODES-constant
+module SymbolToStatusCode
```

Comment:
I’m not too crazy about having this module as part of rubocop-rspec – it doesn’t really belong here…

Would it be possible to hook into Rails or Rack to get this mapping, or would that be too messy? I’m thinking something like
```rb
begin
  require 'rack/utils'
  AUTOCORRECTS = true
rescue LoadError
  AUTOCORRECTS = false
end
```